server/core/src/main/scala/sttp/tapir/server/interceptor/exception/ExceptionHandler.scala

```diff
@@ -1,19 +1,20 @@
 package sttp.tapir.server.interceptor.exception
 
 import sttp.model.StatusCode
-import sttp.tapir.{server, _}
+import sttp.monad.MonadError
+import sttp.tapir._
 import sttp.tapir.server.model.ValuedEndpointOutput
 
-trait ExceptionHandler {
-  def apply(ctx: ExceptionContext): Option[ValuedEndpointOutput[_]]
+trait ExceptionHandler[F[_]] {
+  def apply(ctx: ExceptionContext)(implicit monad: MonadError[F]): F[ValuedEndpointOutput[_]]
```

Comment:
I think I would retain the possibility to handle only some exceptions, hence returning `F[Option[ValuedEndpointOutput[_]]]`

Reply:
Yes, you are right. It's possible with rising error from handler, but returning None is more convenient